README.md

```diff
@@ -1,2 +1,75 @@
-# puppeteer-utils
-Utilities and configuration for running puppeteer against WordPress
+# Puppeteer Utilities for WordPress (puppeteer-utils)
+Utilities and configuration for running puppeteer against WordPress.
+
+
+## How to use
+This package has several utilities and configurations that can be used in your package.
+
+### Configurations
+#### Jest Config
+This configuration pulls in several setup scripts as well as enabling screenshots and slack reporting.
+```javascript
+const { jestConfig } = require( 'puppeteer-utils' );
+
+module.exports = jestConfig;
+
```

Comment:
There are some minor indentation problems such as:
- trailing newlines in code blocks
- missing newlines between different content types (such as titles, paragraphs, code blocks) 

Not a blocked though. I can prepare a PR with markdown fixes once it's landed.

Reply:
Made some changes, including adding more detail about usage and some formatting changes.